wiki/osu!_wiki_Contribution_Guide/en.md

```diff
@@ -57,50 +57,52 @@ At this point, you have two choices:
+*Note: You are not limited to the two choices listed above for editing. To keep this guide short and simple, these articles will not discuss using other tools in-depth or at all. There are other third party applications that can do more or less than what the GitHub Web Interface and GitHub Desktop already does.*
 
 **When you have completed either *GitHub Web Interface* or *GitHub Desktop*, you can continue to the next section.**
 
 ## Finishing
 
 ### Opening a pull request
 
-1\. Go to the [`ppy/osu-wiki` repo](https://github.com/ppy/osu-wiki).
+1. Go to the [`ppy/osu-wiki` repo](https://github.com/ppy/osu-wiki).
 
-2\. If you were quick enough, you may see this yellow banner.
+1. If you were quick enough, you may see this yellow banner.
 
 ![](img/github-recent.jpg)
 
-3\. If you see it, click on the `Compare & pull request` button (skip to step 8). If not, click the `New pull request` button (continue to step 4).
+3. If you see it, click on the `Compare & pull request` button (skip to step 8). If not, click the `New pull request` button (continue to step 4).
 
-4\. On the next page, if you see these two buttons as pictured below, click the `compare across forks`.
+1. On the next page, if you see these two buttons as pictured below, click the `compare across forks`.
 
 ![](img/compare-across-forks-no.jpg "No good.")
 
-5\. Click on the `head fork` dropdown and select the one with your username (it should be the second one).
+5. Click on the `head fork` dropdown and select the one with your username (it should be the second one).
 
 ![](img/head-fork.jpg)
 
-6\. Click on the `compare` dropdown and select the one with the branch you had created (these are listed alphabetically).
+6. Click on the `compare` dropdown and select the one with the branch you had created (these are listed alphabetically).
 
 ![](img/compare-branch.jpg)
 
-7\. Click `Create pull request`.
+7. Click `Create pull request`.
 
-8\. Enter the title in English. The common title naming scheme is the two-lettered language name in square brackets, followed by the article's title. For example, `[FR] BBCode` would mean it is the French version of the BBCode article.
+1. Enter the title in English. This should be a very brief explanation of what you changed.
 
-9\. Fill in the description box with the requested information, you can delete sections if they do not apply.
+  For article translations, include the two-letter language name of your translations in brackets before the title. Your title can just be the name of the article(s) that you are translating. For example, `[FR] BBCode` would indicate that you are updating the French translation of the [BBCode article](/wiki/BBCode).
```

Comment:
Is the indent necessary? Also is it supposed to be separated from number 8?
i.e.
![](https://puu.sh/ADm9G/a6af74d22d.png)

Reply:
the two spaces make the next paragraph grouped with #8 so that it's indented the same as the list. I'm assuming this works on osu-web due to how images behaved (before I removed their indentation)